people-and-planet-ai/geospatial-classification/README.ipynb

```diff
@@ -0,0 +1,1440 @@
+      ],
+      "text/plain": [
+       "<folium.folium.Map at 0x7f4dfbbc20d0>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import folium\n",
+    "import folium.plugins as folium_plugins\n",
+    "import branca.colormap as cm\n",
+    "\n",
+    "colormap = cm.LinearColormap(colors=['lightblue', 'red'], index=[0,1], vmin=0, vmax=1)\n",
+    "map = folium.Map([43.45, -7.87], zoom_start=14, tiles='CartoDB positron')\n",
```

Comment:
Can we visualize satellite images for the tiles? I found the ESRI has freely available ones that we can use.

```
map = folium.Map(
    location=[43.44, -7.86],
    zoom_start=16,
    tiles='https://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}',
    attr = 'ESRI'
)
```

I also slightly modified the coordinates and zoom level to have the power plant more centered.